src/main/test/java/com/playtika/homework6/helper/OpenUrl.java

```diff
@@ -0,0 +1,13 @@
+package com.playtika.homework6.helper;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class OpenUrl {
```

Comment:
It should be `DriverManager` class which returns necessary driver. 'Open any page' activity should be directly in the test